package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "jest",
-    "test:ci": "yarn test --ci",
+    "test:ci": "yarn test --ci --runInBand",
```

Comment:
Halved the jest runtime on CI

ref: https://jestjs.io/docs/en/troubleshooting.html#tests-are-extremely-slow-on-docker-and-or-continuous-integration-ci-server